cmd/init.go

```diff
@@ -0,0 +1,277 @@
+		}
+		fmt.Println("Git initialized in workspace!")
+	}
+
+	// Add GitOps worktree
+	gitopsWorktree := gitopsConfig + "/gitops"
+	worktreeAddCom := exec.Command("git", "worktree", "add", "--orphan", "-b", gitopsName, gitopsWorktree)
+	worktreeAddCom.Dir = gitopsWorkspace
+
+	fmt.Println("Setting up GitOps worktree...")
+	if err := worktreeAddCom.Run(); err != nil {
+		return fmt.Errorf("failed to create GitOps worktree: %w", err)
+	}
+
+	if o.remoteRepo != "" {
+		// Create empty commit
+		emptyCommitCom := exec.Command("git", "commit", "--allow-empty", "-m", "Initial commit")
+		emptyCommitCom.Dir = gitopsWorktree
+		if err := emptyCommitCom.Run(); err != nil {
+			return fmt.Errorf("failed to create empty commit: %w", err)
+		}
+
+		// Push to remote
+		setUpstreamCom := exec.Command("git", "push", "-u", "origin", gitopsName)
+		setUpstreamCom.Dir = gitopsWorktree
+		if err := setUpstreamCom.Run(); err != nil {
+			return fmt.Errorf("failed to set upstream: %w", err)
+		}
+	}	
+
+	fmt.Println("GitOps worktree set up successfully!")
+
+	gitopsLatestVersion, err := dockerhub.GetLatestDockerHubVersion("https://hub.docker.com/v2/repositories/bitswan/gitops/tags/")
+	if err != nil {
+		return fmt.Errorf("failed to get latest BitSwan GitOps version: %w", err)
+	}
+
+	bitswanEditorLatestVersion, err := dockerhub.GetLatestDockerHubVersion("https://hub.docker.com/v2/repositories/bitswan/bitswan-editor/tags/")
+	if err != nil {
+		return fmt.Errorf("failed to get latest BitSwan Editor version: %w", err)
+	}
+
+	
+	createDockerNetworkCom := exec.Command("docker", "network", "create", "bitswan_network")
+
+	fmt.Println("Creating BitSwan Docker network...")
+	createDockerNetworkCom.Run()
+	fmt.Println("BitSwan Docker network created!")
+
+
+	fmt.Println("Setting up GitOps deployment...")
+	gitopsDeployment := gitopsConfig + "/deployment"
+	if err := os.MkdirAll(gitopsDeployment, 0755); err != nil {
+		return fmt.Errorf("Failed to create deployment directory: %w", err)
+	}
+
+	err = caddyapi.AddCaddyRecords(gitopsName, o.domain, o.certsDir != "")
+	if err != nil {
+		return fmt.Errorf("Failed to add Caddy records: %w", err)
+	}
+
+	compose, err := dockercompose.CreateDockerComposeFile(
+		gitopsConfig,
+		gitopsName,
+		gitopsLatestVersion,
+		bitswanEditorLatestVersion,
+		o.certsDir,
+	)
+	if err != nil {
+		return fmt.Errorf("Failed to create docker-compose file: %w", err)
+	}
+
+	dockerComposePath := gitopsDeployment + "/docker-compose.yml"
+	if err := os.WriteFile(dockerComposePath, []byte(compose), 0644); err != nil {
+		return fmt.Errorf("Failed to write docker-compose file: %w", err)
+	}
+
+	err = os.Chdir(gitopsDeployment)
+	if err != nil {
+		return fmt.Errorf("Failed to change directory to GitOps deployment: %w", err)
+	}
+
+	fmt.Println("GitOps deployment set up successfully!")
+
+	projectName := gitopsName + "-site"
+	dockerComposeCom := exec.Command("docker-compose", "-p", projectName, "up", "-d")

```

Comment:
this should be "docker" "compose" `docker-compose` is deprecated.

Reply:
Fixed